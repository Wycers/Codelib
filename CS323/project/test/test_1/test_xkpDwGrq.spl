int woainikanbudao()
{
    int 2, 1_right_id_2;
    float 3_wrong_id;
}