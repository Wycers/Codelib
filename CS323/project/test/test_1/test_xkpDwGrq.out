Error type A at Line 3: unknown lexeme 2
Error type A at Line 3: unknown lexeme 1_right_id_2
Error type A at Line 4: unknown lexeme 3_wrong_id
